@shared/api/internal.ts

```diff
@@ -1358,28 +1490,41 @@ export const loadSettings = (): Promise<
     ExperimentalFeatures & { assetsLists: AssetsLists }
 > =>
   sendMessageToBackground({
+    activePublicKey: null,
     type: SERVICE_TYPES.LOAD_SETTINGS,
   });
```

Comment:
I guess user settings is account agnostic, is that why we're omitting it here?

Reply:
ah, scrape that. I think it's because we don't need the `activePublicKey` to `read` data, we'll only need it to `change/update` data to ensure we're not messing things up with old active accounts. So since this function is a "reader" it should be fine to omit it. Is my thought correct?

Reply:
Yup, that's correct. Really the thing is that `settings` are the same across all accounts. They are not divided by account, so loading settings will always be the same no matter the active public key